modules/sandboxed-containers-create-kata-config-resource-cli.adoc

```diff
@@ -24,6 +24,17 @@ Kata is installed on all worker nodes by default. If you want to install `kata`
 * You have access to the cluster as a user with the `cluster-admin` role.
 * You have installed the {sandboxed-containers-operator}.
 
+[IMPORTANT]
+====
+Creating the `KataConfig` CR automatically reboots the worker nodes. The reboot can take anywhere from ten minutes to over an hour. Factors that impede reboot time are as follows:
+
+* A larger {product-title} deployment with a greater number of worker nodes.
+* Activation of BIOS and Diagnostics.
```

Comment:
is "Diagnostics" supposed to be capitalized?

Reply:
I changed to BIOS and Diagnostics utility.